scripts/dev.sh

```diff
@@ -5,45 +5,21 @@ command_exists() {
   command -v "$1" >/dev/null 2>&1
 }
 
-# Check if jq is installed
-if ! command_exists jq; then
-  echo "jq is not installed. Installing..."
-  sudo apt-get update
-  sudo apt-get install -y jq
-fi
-
 # Check if bos-loader is installed
 if ! command_exists bos-loader; then
     # Install bos-loader
     echo "bos-loader is not installed. Installing..."
     curl --proto '=https' --tlsv1.2 -LsSf https://github.com/mpeterdev/bos-loader/releases/download/v0.7.1/bos-loader-v0.7.1-installer.sh | sh
 fi
 
-# Define default values
-NETWORK_ENV="testnet"
-CREATOR_REPL="REPL_DEVHUB"
-CONTRACT_REPL="REPL_DEVHUB_CONTRACT"
-REPLACE_ACCOUNT=false
-REPLACE_CONTRACT=false
+ACCOUNT_ID="devhub.near"
 
 # Parse command line arguments
 while [[ $# -gt 0 ]]; do
     key="$1"
     case $key in
         -a|--account)
-            ACCOUNT_ID="$2"
-            REPLACE_ACCOUNT=true
-            shift
-            shift
-            ;;
-        -c|--contract)
-            CONTRACT_ID="$2"
-            REPLACE_CONTRACT=true
-            shift
-            shift
-            ;;
-        -n|--network)
-            NETWORK_ENV="$2"
+            ACCOUNT_ID="$2"   
```

Comment:
@Tguntenaar 

Getting rid of the custom contract and network has made the following command stop working:

```cmd
npm run dev:testnet -- -a devhub-dev.testnet -c bodevhub.testnet -n dev
```

The account should be custom because it can either be devhubtest.testnet or devhub-dev.testnet (so package.json should not hardcode the account)

Contract needs to be custom, for example when using bo's contract for announcements

Network should be custom to follow contract and account

Reply:
Hey Elliot! Thank you for pointing this out. We should discuss this. 

The replacement of those constants in now handled in replace.js instead.

The bos-loader only uses the the ACCOUNT_ID to deploy the widgets.

So when you want to change the account / contract / network this can be done by either editing the package.json or making use of the replacements.*.json files.

If we want `npm run dev` to take those arguments we will have to pass it on to the `npm run watch` script, which has to pass it to the `npm run build`, which passes it to the replace... I thought this would be less optimal.

Btw this is because the watch has to rerun the replace script for every build.